app/config/routing_dev.yml

```diff
@@ -6,5 +6,9 @@ _profiler:
     resource: "@WebProfilerBundle/Resources/config/routing/profiler.xml"
     prefix:   /_profiler
 
+_errors:
```

Comment:
@javiereguiluz What about to add an additional route in order to debug HTTP errors like in [symfony-standard](https://github.com/symfony/symfony-standard/blob/master/app/config/routing_dev.yml#L9-L11)?